nix/devshell.nix

```diff
@@ -20,8 +20,9 @@ pkgs.devshell.mkShell ({
       category = "tests";
       command = ''
         export NIX_PATH=nixpkgs=${pkgs.path}
+        # Why? I don't know
+        rm -rf $HOME/.julia/registries/General.toml
```

Comment:
Since Julia 1.7, a mixture of the following registry cases are supported:

- registry dir: `~/.julia/registries/General`
- registry tarball: `~/.julia/registries/General.tar.gz` (Cref: 
https://github.com/JuliaLang/Pkg.jl/pull/2431)

https://github.com/JuliaLang/Pkg.jl/blob/8f19250f346094df3c2b8761d7a9fd482ad57a15/src/Registry/registry_instance.jl#L355-L366

Thus if we directly delete `General.toml` here, the General registry is unavailable.

Reply:
I see, thanks